musicname/ze_ffxiv_wanderers_palace_v5_2f.cfg

```diff
@@ -1,16 +1,16 @@
 "music"
 {
-	"ultimate_starringchild.mp3"			"Aimer - StarRingChild"
-	"ultimate_ref_rain.mp3"			"Aimer - Ref:rain"
-	"ultimate_re_i_am.mp3"			"Aimer - Re I Am"
-	"savage_unseen_abyss.mp3"			"FFXIII-2 OST Final Boss Battle Phase 2 vs Jet Bahamut ( Unseen Abyss )"
-	"savage_hollow.mp3"			"Fate/hollow ataraxia - Aimer - Hollow World"
-	"savage_gundam.mp3"			"Hiroyuki Sawano - [Symphonic Suite UC2012] 5th Mov. :Gundam"
-	"savage_copied_city.mp3"			"NieR Automata OST - Copied City (Sequential Mix) v2"
-	"normal_wanderes_palace_theme.mp3"			"FFXIV OST Wanderer's Palace HM Theme"
-	"normal_the_praetorium_theme.mp3"			"FFXIV OST The Praetorium Theme ( Penitus )"
-	"normal_a_fine_death.mp3"			"FFXIV OST A Fine Death ( Mid-Dungeon Boss Theme )"
-	"hard_revenge_twofold.mp3"			"FFXIV OST 3.4 Boss Theme ( Revenge Twofold )"
-	"hard_duelling.mp3"			"FFXIV OST Duelling Theme / Palace of the Dead"
-	"hard_bismarcks_theme.mp3"			"FFXIV OST - Bismarck's Theme"
+    "ultimate_starringchild.mp3"                "Aimer - StarRingChild"
+    "ultimate_ref_rain.mp3"                     "Aimer - Ref:rain"
+    "ultimate_re_i_am.mp3"                      "Aimer - Re I Am"
+    "savage_unseen_abyss.mp3"                   "FFXIII-2 OST - Jet Bahamut (Unseen Abyss)"
```

Comment:
This is something I'm not sure what to do. Should `Final Fantasy` be abbreviated as `FF` as is? These song names use `FFXIV OST` but some other configs do `Final Fantasy XIV OST`. Which one do you prefer? I prefer the latter, although it'll require you to go back and update all the configs.